test-cases/gutenberg/unsupported-block-editing.md

```diff
@@ -24,27 +28,27 @@
 ##### TC002
 
 **Known Issues**
--  **[Android]** The update button is not greyed-out when changes are discarded
+-  **[Android-only]** The update button is not greyed-out when changes are discarded
 
-### Discarded edits are not persisted
+### Ensure discarded edits are not persisted
 
 1. Repeat steps 1 to 5 above
 2. Edit the block content (e.g. update its text)
 3. Tap the Cancel button to discard changes and expect to be taken back to the block editor
 4. Expect the Update button to be greyed-out
-5. Inspect the HTML content of the post and expect it to **not** contain any of the changes
+5. Using HTML mode in the editor, view the HTML content of the post and expect it to **not** contain any of the changes
 
 ##### TC003
 
-### WP.com Business (Atomic) sites are supported
+### User can edit unsupported blocks on regular WP.com Business (Atomic) sites
 
 Repeat steps from TC001 using a post on a WP.com site with a Business Plan (i.e. Atomic site)
 
 ##### TC004
 
 **Known Issues**
-This feature is disabled for Jetpack connected sites.
+- This feature [not available on Jetpack connected self-hosted sites](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2471)
 
-### Self-hosted sites are supported
+### Self-hosted sites are supported (if they are not connected to WordPress via Jetpack)
 
-Repeat steps from TC001 using a post on a .org site (self-hosted)
+Repeat steps from TC001 using a post on a .org site (self-hosted) that does not have Jetpack connected
```

Comment:
Should this and line 45 say we should repeat TC002 in addition to TC001?

Reply:
> Should this and line 45 say we should repeat TC002 in addition to TC001?

I wasn't sure if it was worth adding to this collection of regression tests, but since I was actually doing both these in my tests today, I'd say now it's a good idea to include them — so I've done that now.

> Also, it might be worth noting that if you use the jetpack-block sample block from TC001, you won't be able to edit the block even on the web on a self-hosted site without jetpack.

That's a good point, something I forgot to add here. Thanks for bringing it up!

I've tweaked a few sentences for clarity and it's ready again for review.